README.md

```diff
@@ -2,6 +2,18 @@
 
 Substra end-to-end (e2e) tests
 
+# A few words regarding tests design
+
+- Must be fast to run
+- Should be stable: no random failure
+- Require a setup with at least 2 nodes
+- Independent from the backends and the network:
+  - backends and network must be started separately
+  - backends and network can be located locally / cloud, started through docker-compose / kubernetes
```

Comment:
Some tests may only work with a specific backend : I'm thinking of adding local data samples. But maybe it's the responsibility of the test to auto-skip itself in case it the env doesn't have the required characteristics.

Reply:
For this specific case yes, and I agree it should be skipped. Currently they don't depend on the backend infrastructure, that could be the case later.

Reply:
👍